arches_querysets/models.py

```diff
@@ -539,33 +539,39 @@ def get_cleaned_default_value(node, default_value):
 
         return cleaned_default
 
-    def get_display_interchange_pair(self, node, node_value):
+    def get_value_with_context(self, node, node_value, datatype_contexts=None):
         datatype_instance = DataTypeFactory().get_instance(node.datatype)
         empty_values = (None, "", '{"url": "", "url_label": ""}')
```

Comment:
I'm not sure if empty curly braces ever show up as "null" values

Reply:
I'll rename this variable to clarify it's just checking display values. Hopefully we aren't using `{}` as display values strings anywhere. There's a follow-up issue filed for fixing URL datatype in this respect.

Reply:
Ok, got it.  I looked over the code and it doesn't look like we're returning {} for any display value.   I just thought maybe it might slip through with a few of the datatypes, but no.